datalad_ria/conftest.py

```diff
@@ -6,4 +6,6 @@
 
 from .tests.fixtures import (
     ria_sshserver,
+    ria_sshserver_setup,
+    common_init_opts,
```

Comment:
```suggestion
    common_ora_init_opts,
```

We better be more specific. Many things get initialized.

Reply:
indeed, I've changed it everywhere in a separate commit